osu.Game/Online/API/Requests/Responses/CommentBundle.cs

```diff
@@ -44,8 +44,40 @@ public List<Comment> Comments
         [JsonProperty(@"user_follow")]
         public bool UserFollow { get; set; }
 
+        private List<Comment> includedComments;
+
         [JsonProperty(@"included_comments")]
-        public List<Comment> IncludedComments { get; set; }
+        public List<Comment> IncludedComments
```

Comment:
I'm confused - what is `IncludedComments` and how does it differ from `Comments`? Why does `IncludedComments`'s getter modify `Comments` too?

Reply:
`IncludedComments` contains all non-top-level comments (in case of `GetRepliesRequest`)